### DISCUSSION TITLE
Can I use it to synthesis an HEIF HDR image using HDR AVIF or RAW?

### DISCUSSION THREAD

**Discussion by TIMESTICKING** [awxkee/avif-coder#22] [Q&A] [answered]
I'd like to create shareable HDR images using RAW files, at the first time I tried to export the images into AVIF, but I found that Apple does not support viewing HDR AVIF files, however, Apple made their own standards, i.e. HEIF with gain map, so can I use this tool to generate HDR HEIF so that I can share my HDR photos and view them on my phone?

If it can't, are there any tools that can do this?

**Comment by awxkee** [comment, marked as answer]
This tool works in android ecosystem, and can export HDR AVIF however at first you'll need to load android bitmap with suitable profile what can be tricky enough.

Apple do support basic HDR support in AVIF files using PQ and HLG transfer functions. I think fffmpeg should support raw files, if not ImageMagick also possible a solution, avifenc ( this is a part of libavif ) can also might be solution however I'm not sure this tool support RAW files. Also preview tool on mac can set Rec.2020 PQ profile.
You'll need to set color profile Rec.2020 and transfer function PQ.
